### ISSUE TITLE
Merge graphs dark storage performance benchmark

### ISSUE THREAD

**Issue by jonathan-eq** [equinor/ert#6966]
Dark storage performance benchmark provides 40 different graphs on the gh-pages dashboard. Have them merged into one. 
The graph should show the previous 5 commits in addition to the mean

**Comment by jonathan-eq** [comment]
Blocked by not having enough data to experiment with this. Will let the workflow run for a couple of days before continuing with this.

**Comment by jonathan-eq** [comment]
![image](https://github.com/equinor/ert/assets/107626001/61730946-5a4a-4c95-8f56-fd7f66d10d0a)

**Comment by jonathan-eq** [comment]
Partially solved. I had to change to a custom script to generate the svg with matplotlib, because the commit ids were incorrect for the marketplace action. The entire workflow can now be rewritten to only run in komodo-releases, and not be split across two repos.

**Comment by jonathan-eq** [comment]
https://equinor.github.io/ert/dev/dark_storage_performance_benchmark/benchmark.svg

**Comment by jonathan-eq** [comment]
The graphs have been merged. A future PR could be to improve the plot, but the initial plot is finished.
